.github/flash_talos.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+TALOS_IP=192.168.20.9
+TALOS_USER=root
+LOG_FILE=/dev/null
+
+COREBOOT_SOURCE=build/coreboot.rom.signed.ecc
+COREBOOT_BMC_LOCATION=/tmp/coreboot.rom.signed.ecc
+COREBOOT_PARTITION=HBI
+
+BOOTBLOCK_SOURCE=build/bootblock.signed.ecc
+BOOTBLOCK_BMC_LOCATION=/tmp/bootblock.signed.ecc
+BOOTBLOCK_PARTITION=HBB
+
+echo "Power off Talos II"
+ssh $TALOS_USER@$TALOS_IP obmcutil -w poweroff > $LOG_FILE
+
+echo "Copy coreboot image"
+scp $COREBOOT_SOURCE $TALOS_USER@$TALOS_IP:$COREBOOT_BMC_LOCATION > $LOG_FILE
+scp $BOOTBLOCK_SOURCE $TALOS_USER@$TALOS_IP:$BOOTBLOCK_BMC_LOCATION > $LOG_FILE
```

Comment:
If the goal was to log to a file (e.g. for debugging), then this will work porly, as each command executions overwrites the file entirely (not appending).

Reply:
In that case we should probably redirect stderr, too.

Reply:
The idea was to simply silent the output of the commands, but thought to myself, it would be better to provide an easy way to enable or redirect it. I applied Your suggestions and now the log file is appended and stderr is redirected too.